### ISSUE TITLE
Optimize placement of controls

### ISSUE THREAD

**Issue by probonopd** [flaviotordini/minitube#231]
The placement of controls could be improved to make the application much more usable and in line with conventions:

* The search box should be placed above the search results
* The media player controls should be placed underneath the media player, as in >99% of all applications

This would be a huge improvement in usability.

![image](https://user-images.githubusercontent.com/2480569/204125901-56eadaf9-9bc6-465c-ac8b-931aca1936f8.png)

Thanks for your consideration.

**Comment by chrisgoldberg1** [comment]
The benefit of it <-> the work for it != worth it

**Comment by probonopd** [comment]
Yes, small fixes that make a huge difference in terms of usability!

**Comment by chrisgoldberg1** [comment]
Maybe my message was not so clear. Its a lot of work to position these elements like you suggested. The current concept of code is not made for it and the functionality of these elements is given. 
I doubt someone will spend a few days or even weeks for it. Its not worth it.
